lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,12 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.nn.ru/"],
```

Comment:
I do not see nn.ru in list of allowed resources

Reply:
all hybrid websites presented in the table are taken, am i allowed to work with one of them despite this fact? we have agreed that for zebra-tv.ru recursive crawler would not work